### ISSUE TITLE
Support for Component Router - Angular 1

### ISSUE THREAD

**Issue by sebastiannm** [angulartics/angulartics#467]
Hi,

I was just trying out this module with the new component router and  it brakes the app.

NgComponentRouter has $route defined but it's null. So line 170 ($injector.has('$route')) returns true but the value of $route is null, which makes the loop coming after to brake.

Any plans on supporting this new Angular feature?

**Comment by timelf123** [comment]
I haven't had a chance to play with that yet. Is it as simple as adding a null check to another if block and handling component router logic there?

**Comment by sebastiannm** [comment]
@timelf123 I did that on my side and I have been playing with it for 3 hours without errors.

I also looked at what I send to analytics and it works correctly.

```
if ($route) {
          for (var route in $route.routes) {
            noRoutesOrStates = false;
            break;
          }
        } else if ($route === null){
          noRoutesOrStates = false;
        }
```

**Comment by timelf123** [comment]
@sebastiannm would you mind rolling a PR so I can review later?

**Comment by sebastiannm** [comment]
@timelf123 Yes I created one now: https://github.com/angulartics/angulartics/pull/468
